sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Hotspots/DoNotHardcodeCredentialsBase.cs

```diff
@@ -155,6 +157,14 @@ private IEnumerable<string> FindCredentialWords(string variableName, string vari
                 // before reporting to avoid new issues to appear on SQ/SC.
                 return credentialWordsFound.Select(x => x.ToLowerInvariant());
             }
+
+            private bool IsValidParameter(string suffix)
+            {
+                var candidateParameter = suffix.Split(';').First().Trim();
```

Comment:
Is `candidateParameter` a good name? I would suggest to use `password` to see what is known so far. It will be parameter after the regex check.

Reply:
Indeed it was a bit strange. I've rename it to credential to match the "CredentialWords" property (since the user can customize in the configuration the CredentialWords)